charts/vela-core/templates/defwithtemplate/suspend.yaml

```diff
@@ -1,16 +0,0 @@
-# Code generated by KubeVela templates. DO NOT EDIT. Please edit the original cue file.
-# Definition source cue file: vela-templates/definitions/internal/suspend.cue
-apiVersion: core.oam.dev/v1beta1
-kind: WorkflowStepDefinition
-metadata:
-  annotations:
-    definition.oam.dev/description: Suspend your workflow
-  name: suspend
-  namespace: {{ include "systemDefinitionNamespace" . }}
-spec:
-  schematic:
-    cue:
-      template: |
-        // no parameters
-        parameter: {}
-
```

Comment:
why this file deleted?

Reply:
Sorry, file is recreated automatically and not added via git cmd. I'll commit it.